disperser/common/blobstore/blob_metadata_store.go

```diff
@@ -49,6 +54,13 @@ func (s *BlobMetadataStore) QueueNewBlobMetadata(ctx context.Context, blobMetada
 		return err
 	}
 
+	if s.shadowTableName != "" && s.shadowTableName != s.tableName {
```

Comment:
can we add a test that items are replicated in the shadow table in the existing test cases? i.e. `shared_store_test.go` and `server_test.go`